box-share-sdk/src/main/res/values/strings.xml

```diff
@@ -13,7 +13,11 @@
     <string name="box_sharesdk_accessible_collaborator" description="description for collaborator access">Accessible to people the folder is shared with</string>
     <string name="box_sharesdk_password_protected" description="description for if a shared link is password proceted">Password protected</string>
     <string name="box_sharesdk_password_unprotected" description="description for if a shared link is not password proceted">No password</string>
-    <string name="boxsdk_fetching_collaborators" description="message to show on spinner when we are fetching collaborators">Fetching collaborators</string>
+    <string name="box_sharesdk_fetching_collaborators" description="message to show on spinner when we are fetching collaborators">Fetching collaborators</string>
+    <string name="box_sharesdk_adding_collaborators" description="message to show on spinner when we are adding collaborators">Adding collaborators</string>
+    <string name="box_sharesdk_enabling_share_link" description="message to show on spinner when we are enabling share link">Enabling share link</string>
```

Comment:
I am trying to be consistent with the wording we use in our app. We use the term "Share Link" everywhere so I thought in the message "Enabling share link" would be more accurate.